server/src/database/config/fakeData.sql

```diff
@@ -0,0 +1,7200 @@
+INSERT INTO users (name, email, password)
+VALUES (
+        'احمد',
+        'dlouys0@angelfire.com',
+        '$2b$10$D7oI89od5EhIzJnB.cCgmO5pneIeynM27potj535oATgM3W01m4XS'
+    );
+INSERT INTO users (name, email, password)
```

Comment:
No need to write the command every insertion 
It is better to write in this way
```
INSERT INTO users(name, email, password) VALUES (), (), ();
```

Reply:
This data is inserted by default from mockroo.com,
It will be difficult to change all the fake data, and anyway, It works normally.